src/models/db/account.js

```diff
@@ -54,6 +54,7 @@ class Account extends Model {
     delete data.primary
     delete data.password_hash
     delete data.public_key
+    delete data.fingerprint
```

Comment:
The fingerprint is a hash of the contents of the client certificate. I don't think there's any case where this needs to be communicated back to a client